packages/material-ui/src/SvgIcon/SvgIcon.js

```diff
@@ -48,6 +54,7 @@ function SvgIcon(props) {
 
   const className = classNames(
     classes.root,
+    classes[`fontSize${capitalize(fontSize)}`],
```

Comment:
Can be simplified following your previous comment.

Reply:
you mean like we do with color or sth more static like: `classes.fontSizeInherit: fontSize === 'inherit'`?